src/probe.rs

```diff
@@ -15,7 +19,7 @@ pub(crate) fn open(opts: &cli::Opts) -> Result<Probe, anyhow::Error> {
     };
 
     if filtered_probes.is_empty() {
-        bail!("no probe was found")
+        bail!("{}", NO_PROBE_FOUND_ERR)
```

Comment:
Why is there no leading "Error" as in line 50?

Reply:
Because `bail!()` adds the "Error: " prefix automatically (that's why I added it in line 50 as well, for consistency)